Cargo.toml

```diff
@@ -9,13 +9,16 @@ documentation = "https://docs.rs/evdev"
 edition = "2021"
 
 [features]
+serde = ["serde_1"]
 tokio = ["tokio_1", "futures-core"]
 
 [dependencies]
 libc = "0.2.121"
 bitvec = "1.0.0"
 nix = "0.23"
+paste = "1.0"
```

Comment:
`paste` is only needed for feature `serde`, right?

Reply:
Yes, I had that originally, but then ran into issues with the macro itself requiring `paste::paste!` to be conditionally invoked based on the serde feature flag, and I am not sure how to do that.